src/components/measureList/MeasureList.tsx

```diff
@@ -280,7 +360,15 @@ export default function MeasureList(props: {
                           measure?.version,
                           measure?.revisionNumber
                         )}
-                        <Chip tw="ml-6" className="chip-draft" label="Draft" />
+                        <Chip
+                          tw="ml-6"
+                          className="chip-draft"
+                          label={
+                            `${measure.measureMetaData?.draft}` === "true"
+                              ? "Draft"
+                              : "Non-Draft"
```

Comment:
Do we want to display "Non-Draft" to users on list page? I couldn't find any designs for this.

Reply:
@RohitKandimalla I removed "Non-Draft", thanks!